Jenkinsfile

```diff
@@ -0,0 +1,74 @@
+pipeline {
+    agent {
+        node 'linux-ci'
```

Comment:
minor, instead of straight up specifying `linux-ci`, it's better to use the node tags, so in this case it would be
```
agent {
    label 'linux && python'
}
```

note: I'm not 100% certain this is the exactly format but we should be able to specify labels.

see https://jenkins.io/doc/book/pipeline/jenkinsfile/#using-multiple-agents

Reply:
labels are given here: https://ci.rev.com/computer/linux-ci/